Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -1590,8 +1590,24 @@ open class NavigationMapView: UIView {
                 navigationCameraType: NavigationCameraType = .mobile,
                 tileStoreLocation: TileStoreConfiguration.Location? = .default) {
         super.init(frame: frame)
-        
-        setupMapView(frame, navigationCameraType: navigationCameraType, tileStoreLocation: tileStoreLocation)
+        let mapView = makeMapView(
+            frame: frame,
+            navigationCameraType: navigationCameraType,
+            tileStoreLocation: tileStoreLocation
+        )
+        setupMapView(mapView: mapView, navigationCameraType: navigationCameraType)
+        commonInit()
+    }
+
+    /// :nodoc:
+    public init(
```

Comment:
If you do not plan to make this initializer documented it makes sense to remove changelog entry for now.

Reply:
Also would be great to have a unit-test for this case.